sleap_roots/points.py

```diff
@@ -190,14 +219,26 @@ def get_root_vectors(start_nodes: np.ndarray, end_nodes: np.ndarray) -> np.ndarr
         An array of vectors with shape (instances, 2), representing the vector from start
         to end for each instance.
     """
+    # If start or end nodes is not a 2D array, make it one
+    start_nodes = np.atleast_2d(start_nodes)
```

Comment:
after this shouldn't dimension of arrays both be 2? can we use that to simplify the logic?